doc/release-process.md

```diff
@@ -375,6 +375,8 @@ bitcoin.org (see below for bitcoin.org update instructions).
 
   - Bitcoin Core Twitter https://twitter.com/bitcoincoreorg
 
+  - Update the title of the #bitcoin IRC channel
```

Comment:
I don't think anyone of us can change the title of an unaffiliated channel

Reply:
Ah ok, thanks--updated.